### PR TITLE
Make auth path configurable via env

### PR DESCRIPTION
**Pull request by concaf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Example auth service for [Ambassador][ag] using [ExtAuth][ae]. See the [Ambassad
 
 The service listens on port 3000 and supports the ExtAuth interaction pattern. In short, the service expects a POST to the path `/ambassador/auth` with client request headers as a JSON map in the POST body. If auth is okay, it returns a 200 to allow the client request to go through. Otherwise it returns a 401 with an HTTP Basic Auth WWW-Authenticate header. Take a look at [ExtAuth][ae] for more information about the Envoy side of things.
 
-The current version of this code only performs auth when the client headers indicate a request under the `/service` path. The only valid credentials are `username:password`. It should be straightforward to extend this sample to perform meaningful auth.
+By default, this code only performs auth when the client headers indicate a request under the `extauth/backend/get-quote` path. This path can be tweaked via the `AUTH_PATH` environment variable. The only valid credentials are `username:password`. It should be straightforward to extend this sample to perform meaningful auth.
```
Comment (ark3) [inline README.md]: This implies the default is `extauth/backend/get-quote` but the default is actually `/extauth/backend/get-quote` (with the leading slash). I suggest including the leading slash here.

Comment (concaf) [reply README.md]: Fixed, thanks!

File: server.js [hunk 2]
```diff
@@ -40,8 +40,14 @@ app.use(addRequestId)
 // Add verbose logging of requests (see below)
 app.use(logRequests)
 
-// Require authentication for /extauth/backend/get-quote/ requests
-app.all('/extauth/backend/get-quote*', authenticate, function (req, res) {
+// Get authentication path from env, default to /extauth/backend/get-quote
+var authPath = '/extauth/backend/get-quote'
+if (process.env.AUTH_PATH) {
+  authPath = process.env.AUTH_PATH
+}
```
Comment (ark3) [inline server.js:47]: I think output here would be helpful.
```javascript
console.log(`setting authenticated path to ${authPath}`)
```
Also might be good to update line 66 below, which still says `/backend/get-quote`.

Comment (concaf) [reply server.js:47]: Makes sense. Fixed, thanks!

**Review by ark3 — APPROVED** [review]
lgtm (I assume it works! :smile:)

**Pull request merged**